src/engine/abstract/ARStaffFormat.cpp

```diff
@@ -101,6 +102,13 @@ void ARStaffFormat::setTagParameterList(TagParameterList & tpl)
 				fLineThickness = (LSPACE * fval->getValue() >= 0 ? LSPACE * fval->getValue() : 0);
 
 			delete fval;
+
+            TagParameterFloat* distance = TagParameterFloat::cast(rtpl->RemoveHead());
+            assert(distance);
+            if (distance->TagIsSet())
+                fDistance = (LSPACE * distance->getValue() >= 0 ? LSPACE * distance->getValue() : 0);
```

Comment:
don't assume the value is a LSPACE, conversion will be automatic if you're using a unit type